plugins/modules/group.py

```diff
@@ -174,7 +183,10 @@ def modify_group(module):
                 opts += f"{ attr }={ val } "
         if load_module_opts is not None:
             opts = load_module_opts + opts
-        cmd = f"chgroup { opts } { name }"
+
+        if module.params['load_module']:
```

Comment:
This condition is not needed as load_module always will have 'files' as default value and because it is part of choice, any other value will be handled by module as invalid.